### ISSUE TITLE
Requirements ?

### ISSUE THREAD

**Issue by walter-weinmann** [rfrancis/asdf-gradle#9]
What are the installation requirements with a fresh Ubuntu 20.04?

```
export GRADLE_HOME=/opt/gradle/gradle-7.1.1
/home/walter/.asdf/plugins/gradle/bin/install: 10: Syntax error: redirection unexpected

```

**Comment by rfrancis** [comment]
Looks like it uses a redirect that dash (which is what /bin/sh links to on Ubuntu 20.04) doesn't support. Probably we should go ahead and specify /bin/bash if that's what we mean.

You might try editing the file to say #!/bin/bash in the first line and see if that fixes things for you and let us know.

**Comment by walter-weinmann** [comment]
My script already starts with `#!/bin/bash`:

```
#!/bin/bash

set -e

# ------------------------------------------------------------------------------
# Install Gradle.
# ------------------------------------------------------------------------------
# Prerequisites
# =============
# asdf
# ------------------------------------------------------------------------------

cd ~/

# Setting Environment Variables ------------------------------------------------
export DEBIAN_FRONTEND=noninteractive
export LOG_FILE=gradle_2.log

export VERSION_GRADLE=7.1.1

exec &> >(tee -i ${LOG_FILE}) 2>&1
sleep .1

echo "================================================================================"
echo "Start $0"
echo "--------------------------------------------------------------------------------"
date +"DATE TIME : %d.%m.%Y %H:%M:%S"
echo "================================================================================"
echo "Supplement necessary system software"
echo "--------------------------------------------------------------------------------"
sudo apt-get clean -qy
sudo apt-get update -qy
sudo apt-get upgrade -qy

sudo apt-get install -qy curl

sudo rm -rf ~/.asdf/downloads/gradle
sudo rm -rf ~/.asdf/installs/gradle
sudo rm -rf ~/.asdf/plugins/gradle

echo "================================================================================"
echo "Install required software"
echo "--------------------------------------------------------------------------------"
export GRADLE_HOME=/opt/gradle/gradle-${VERSION_GRADLE}
eval echo 'export GRADLE_HOME=/opt/gradle/gradle-${VERSION_GRADLE}'
eval echo 'export PATH=${GRADLE_HOME}/bin:${PATH}' >> ~/.bashrc

asdf plugin add gradle
asdf list all gradle
asdf install gradle ${VERSION_GRADLE}
asdf global gradle ${VERSION_GRADLE}

echo " "
echo "=====================================================================> Version  asdf: "
echo " "
echo "Current version of asdf: $(asdf --version)"
echo " "
echo "=====================================================================> Version  Gradle: "
echo " "
echo "Current version of Gradle: $(gradle --version)"
echo " "
echo "=====================================================================> Environment variable GRADLE_HOME: "
echo " "
echo ${GRADLE_HOME}
echo " "
echo "--------------------------------------------------------------------------------"
date +"DATE TIME : %d.%m.%Y %H:%M:%S"
echo "--------------------------------------------------------------------------------"
echo "End   $0"
echo "================================================================================"

exit 0

```

**Comment by rfrancis** [comment]
That looks like something of yours. I'm talking about the install script installed as part of the plugin, which in your case is at ```/home/walter/.asdf/plugins/gradle/bin/install```

**Comment by walter-weinmann** [comment]
Ok, I understood and changed it, but unfortunately, it made no difference.

**Comment by rfrancis** [comment]
I can't replicate this on an Ubuntu 20.04 system. Please list the exact set of steps you take that produces this error?  If I do:

```
$ asdf plugin-add gradle
$ asdf install gradle 7.1.1
```

I get no errors and it works as expected.

**Comment by walter-weinmann** [comment]
Please see my script above - thanks.

**Comment by rfrancis** [comment]
I simply can't reproduce this problem on an Ubuntu 20.04 system. The error is obviously about the <<< redirection on line 10 of the install script, but I'm simply not having that problem on either Ubuntu 20.04 system I tried. Strange.

The only thing I can immediately think to do is for me to change the install script to specify /bin/bash like I mentioned above, except to do so in the repo so when your script pulls it down it'll get that version. I have made the change and your script works for me (but then, it already did!)  So you might try it again.
